spec/integration/failed_scheduled_publishing_spec.rb

```diff
@@ -36,6 +37,11 @@ def then_the_edition_is_set_to_failed_to_publish
     expect(@edition.reload).to be_failed_to_publish
   end
 
+  def and_a_timeline_entry_is_created
+    timeline_entry = @edition.timeline_entries.last
+    expect(timeline_entry.entry_type).to eq("scheduled_publishing_failed")
```

Comment:
Would it be possible to do this through the document summary page? I think we conventionally try to avoid expectations on the database in feature tests, since it isn't something users see. And it would help to keep our I18n coverage up ;-). I suppose it's a bit unclear whether an integration test should really be visiting a view - does that sound reasonable to you?

Reply:
That's something that is done through https://github.com/alphagov/content-publisher/pull/1160